libsplinter/src/admin/service/event/store/diesel/schema.rs

```diff
@@ -0,0 +1,114 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+table! {
+    admin_event_entry (id) {
+        id -> Int8,
+        circuit_id -> Text,
+        event_type -> Text,
+        data -> Nullable<Binary>,
+        timestamp -> Timestamp,
+    }
+}
+
+table! {
+    admin_event_circuit_proposal (event_id, circuit_id) {
+        event_id -> Int8,
+        proposal_type -> Text,
+        circuit_id -> Text,
+        circuit_hash -> Text,
+        requester -> Binary,
+        requester_node_id -> Text,
+    }
+}
+
+table! {
+    admin_event_create_circuit (event_id, circuit_id) {
```

Comment:
We should be consistent with naming from the admin service store and call this `admin_event_proposed_circuit`